src/code.py

```diff
@@ -519,31 +471,34 @@ def handle_key_press(key):
         # app_titles[selected].background_color = launcher_config.palette_accent
 
     if mouse:
-        try:
-            # attempt to read data from the mouse
-            # 10ms timeout, so we don't block long if there
-            # is no data
-            count = mouse.read(mouse_endpoint_address, mouse_buf, timeout=20)
-        except usb.core.USBTimeoutError:
-            # skip the rest of the loop if there is no data
-            count = 0
-
-        # update the mouse tilegrid x and y coordinates
-        # based on the delta values read from the mouse
-        if count > 0:
-            mouse_tg.x = max(0, min((display.width // scale) - 1, mouse_tg.x + mouse_buf[1]))
-            mouse_tg.y = max(0, min((display.height // scale) - 1, mouse_tg.y + mouse_buf[2]))
```

Comment:
This code was accounting for the scale that is applied to most of the visible elements. This allowed it to be properly clamped within the visible area. 

With the new version from this PR the mouse is allowed to venture outside of the screen on the right and bottom sides due to the `scale` not being accounted for. 

It's possible to set the scale on the BootMouse object returned by the `find_and_init_boot_mouse()` function i.e.

```
if launcher_config.use_mouse:
    mouse = find_and_init_boot_mouse()
    if mouse:
        mouse.scale = 2
        mouse_tg = mouse.tilegrid
        mouse_tg.x = display.width // (2 * scale)
        mouse_tg.y = display.height // (2 * scale)
```

Reply:
ugh, the monitor I was using isn't 100% fruit jam happy, the image runs off the right and bottom of the screen. Just my luck that's where the code bug was.....

Thanks. I'll switch monitors and then get your fix in....